src/store/tasks/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { ITaskState } from '../../types/tasks';
+import { PayloadAction } from '@reduxjs/toolkit';
+
+export const setNewLabel = (state: ITaskState, action: PayloadAction<string>) => {
+  state.label = action.payload;
+};
+
+export const setSelectedStatus = (state: ITaskState, action: PayloadAction<string>) => {
+  state.status = action.payload;
+};
+
+export const setModalOpen = (state: ITaskState, action: PayloadAction<boolean>) => {
```

Comment:
You need to control modal from local state

Reply:
fix: local modal state